package.json

```diff
@@ -42,10 +47,14 @@
         "prettier": "3.3.3",
         "react": "^18.3.1",
         "react-dom": "^18.3.1",
+        "sass": "^1.80.6",
```

Comment:
this is beyond the context of the PR 😅 , let's be more specific when adding changes next time

Reply:
Got it, I've added sass just for styling the Callback component for now